projects/cm_mcu/Semaphore.h

```diff
@@ -0,0 +1,40 @@
+/*
+ * Semaphore.h
+ *
+ *  Created on: Semptember 06, 2022
+ *      Author: pkotamnives
+ */
+
+#ifndef PROJECTS_CM_MCU_SEMAPHORE_H_
+#define PROJECTS_CM_MCU_SEMAPHORE_H_
+
+#include "FreeRTOS.h"
+#include "semphr.h"
+
+/*
```

Comment:
please delete all the commented out parts that are unused in this header.

Reply:
sorry forgot to clean up. Done.